shared/domains/chelonia/SPMessage.js

```diff
@@ -290,7 +290,7 @@ export class SPMessage {
     if (!state?._vm?.authorizedKeys && head.op === SPMessage.OP_CONTRACT) {
       const value = rawSignedIncomingData(parsedValue)
       const authorizedKeys = Object.fromEntries(value.valueOf()?.keys.map(wk => {
-        const k = unwrapMaybeEncryptedData(wk.valueOf())
+        const k = unwrapMaybeEncryptedData(wk)
```

Comment:
This is unrelated to workers, but a small change that's needed for the use of `unwrapMaybeEncryptedData` to make sense. `unwrapMaybeEncryptedData` is supposed to take `X | EncryptedData<X>`, and the previous use was only `X`, since `.valueOf()` already unwraps.